lib/node_modules/@stdlib/math/base/assert/is-odd/README.md

```diff
@@ -104,6 +104,97 @@ for ( i = 0; i < 100; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/assert/is_odd.h"
+```
+
+#### stdlib_base_is_odd( x )
+
+Tests if a finite numeric value is an odd number.
+
+```c
+bool out = stdlib_base_is_odd( 1.0 );
+// returns true
+
+out = stdlib_base_is_odd( 4.0 );
+// returns false
+```
+
+The function accepts the following arguments:
+
+-   **x**: `[in] double` input value.
+
+```c
+bool stdlib_base_is_odd( const double x );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/math/base/assert/is_odd.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+int main( void ) {
+    double x;
+    bool v;
+    int i;
+
+    for ( i = 0; i < 100; i++ ) {
+        x = ( ( (double)rand() / (double)RAND_MAX ) * 100.0 );
+        v = stdlib_base_is_odd( x );
+        printf( "x = %lf, is_odd(x) = %s\n", x, ( v ) ? "odd" : "not odd" );
```

Comment:
I suggest following more closely to what we did in https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/assert/is-nan/examples/c/example.c. Meaning, use a set list of values and update the message on L185 to match.

Reply:
I guess he followed https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/assert/is-integer/examples/c/example.c

Reply:
It looks to be a mash-up. I suggest simplifying here, and just using a fixed list of values.